openmicroscopy/openmicroscopy
# Missing icon

While investigating connection error. I noticed that few icons were missing i.e.

```
2013-03-06 14:41:17,344 ERROR [opy.shoola.agents.treeviewer.IconManager] (nitializer) Failed to retrieve icon: <classpath>/org/openmicroscopy/shoola/agents/treeviewer/graphx/tagged_dataset_reloadnotowned16.png.
2013-03-06 14:41:17,350 ERROR [opy.shoola.agents.treeviewer.IconManager] (nitializer) Failed to retrieve icon: <classpath>/org/openmicroscopy/shoola/agents/treeviewer/graphx/nuvola_folder_blue_open_modified_screen_annotated_reload_notowned16.png.

```

This PR addresses the issue.

To test:

Launch insight and check that the lines listed above are not in the insight log.
